test/database.test.ts

```diff
@@ -7,6 +7,7 @@ import { Database } from "../src/database.js";
 import { Statement } from "../src/statement.js";
 
 describe("database", function () {
+  this.timeout("10s");
```

Comment:
Not related to this PR, but the mocha tests are occasionally timing out.  I moved from 5s to 10s in several of the files that were timing out.